ui/src/app/utils/stores.tsx

```diff
@@ -46,6 +46,12 @@ export interface CompoundRuntimeOpStore {
   [workspaceNamespace: string]: CompoundRuntimeOperation;
 }
 
+export interface ServerConfigStore {
```

Comment:
Created this because I didn't want to inject `serverConfigService` into the new `Context`